main.cpp

```diff
@@ -61,11 +60,13 @@ int main(int argc, const char * argv[]) {
     cout << is_permutation("Apple", "Pabble") << "\n";
```

Comment:
Usually, it's better for the tests to actually check the output automatically, and the output of the whole program be only which tests failed (showing inputs, expected and actual outputs), or that all tests pass (and how many where run).
This way, if you break something fixing something else you will notice it and don't have to manually check the results each time.